src/ValueConverters/StringFormatConverter.cs

```diff
@@ -25,32 +20,14 @@ namespace WPFLocalizeExtension.ValueConverters
     /// </summary>
     public class StringFormatConverter : TypeValueConverterBase, IMultiValueConverter
     {
-        private static MethodInfo miFormat = null;
-
         #region IMultiValueConverter
         /// <inheritdoc/>
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (miFormat == null)
-            {
-                try
-                {
-                    // try to load SmartFormat Assembly
-                    var asSmartFormat = Assembly.Load("SmartFormat");
```

Comment:
you can not remove the code, I explain it in the ticket

Reply:
Oh, yes, I've understood. It's an interesting and useful library.
This converter is hard to tes with dynamic invocation, so I think my unit tests will be useless. What would you like me to do with this PR?
* Close this PR without merging
* Rollaback invokaction, test only one calling (string.Format or SmartFormat), merge changes from #282 
* Try refactor class for unit testing for support both calling

There is another idea with breacking changes. I can create `protected virtual string Format(string format, params object[] args)` method, create new `SmartStringFormatConverter` and override just one simple method with dynamic invocation. Also users can create own `SmartStringFormatConverter` with static call.

What do you think?

Reply:
I will have time on the weekend to take care of this topic. I think I will cherrypick from your PR and after everything is in the dev branch we can close your one, but please let them at the moment open.